apps/music/views/albums/view.js

```diff
@@ -36,7 +36,7 @@ var AlbumsView = View.extend(function AlbumsView() {
           return unknown;
         }
 
-        var sectionName = item.metadata.album[0].toLowerCase();
+        var sectionName = item.album[0].toLowerCase();
```

Comment:
Couldn't we just move all of this logic into the `map()` function? Just add a property to the objects called `sectionName`.